### ISSUE TITLE
New default runner that handles :optimizations :none

### ISSUE THREAD

**Issue by slagyr** [ecmendenhall/specljs-template#2]
See: https://github.com/slagyr/speclj/issues/100

**Comment by Jell** [comment]
any progress on that?

**Comment by slagyr** [comment]
Yes.  We haven't got a silver bullet solution but :optimization :none is a solved problem.  In fact, the speclj project uses this.  Have a look.  I'm sure you'll be able to adapt that solution to your project.  You'll notice that there's a handful of changes you'll need to make for your setup. And speclj no longer uses cljsbuild, but you can.

Start with https://github.com/slagyr/speclj/blob/master/bin/speclj.js
This is the new starting point.  The test command becomes `phantomjs specl.js`.
This script can use used as an "auto runner" to run only specs that have been changed since last run.

Then check out https://github.com/slagyr/speclj/blob/master/bin/specs.html
An HTML document is the only way I was able to get :optimization :none source to run right.  But once it's wired up, it's nice because you can run your specs in the browser if desired.

As you work through it, maybe you could suggest a standard project structure so that we can generate this runner for new projects.

**Comment by Jell** [comment]
thanks, I'll have a crack at it next week! Cheers
